packages/airnode-node/src/evm/authorization/authorization-fetching.ts

```diff
@@ -114,7 +115,7 @@ export async function fetch(
   }
 
   // If there are no authorizer contracts then endpoint is public
-  if (isEmpty(fetchOptions.authorizers)) {
+  if (isEmpty(Object.values(fetchOptions.authorizers).flatMap((authorizer) => authorizer))) {
```

Comment:
Since we're passing directly only `requesterEndpointAuthorizers` for the check (comment above) I think we should check only that for emptiness for v0.